server/routes/auth.ts

```diff
@@ -273,7 +273,7 @@ app.use(passport.session());
 export let authRoutes = express.Router();
 
 function getExternalPort(request: express.Request): number {
-	let host = request.headers.host!;
+	let host = request.headers.host as string;
```

Comment:
This is just for the type system and doesn't do any actual conversion of the values. It's required because the type definitions for Node were recently updated so that header values are of the type `string | string[] | undefined`. Since [`Host` headers must always appear](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Host), the value can't be `undefined` and will not be an array.